mappings/net/minecraft/data/server/RecipesProvider.mapping

```diff
@@ -174,8 +183,27 @@ CLASS net/minecraft/class_2446 net/minecraft/data/server/RecipesProvider
 	METHOD method_33555 createSignRecipe (Lnet/minecraft/class_1935;Lnet/minecraft/class_1856;)Lnet/minecraft/class_5797;
 		ARG 0 outputItem
 		ARG 1 inputItem
+	METHOD method_33714 convertBetween (Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)Ljava/lang/String;
+		ARG 0 from
+		ARG 1 to
+	METHOD method_33715 offerRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;I)V
+		ARG 0 exporter
+		ARG 1 output
+		ARG 2 input
+		ARG 3 count
+	METHOD method_33716 getItemPath (Lnet/minecraft/class_1935;)Ljava/lang/String;
+		ARG 0 item
+	METHOD method_33717 offerRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
+		ARG 1 input
+		ARG 2 output
 	METHOD method_34662 offerCrackingRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
 		COMMENT Offers a smelting recipe to the exporter that is used to convert the main block of a block family to its cracked variant.
 		ARG 0 exporter
 		ARG 1 result
 		ARG 2 input
+	METHOD method_34854 offerWaxingRecipes (Ljava/util/function/Consumer;)V
```

Comment:
Make sure to map the lambda parameters for this too.

Reply:
Alrighty, should be sorted now! I also spotted a few methods I missed and parameters that weren't following the convention of the rest of that class.